README.md

```diff
@@ -11,19 +11,18 @@ To run this tool locally you need:
 
 ## Running locally
 
-* Restore the dotnet tools:
+* Perform a one-time set up of source dependencies:
 
-> dotnet tool restore
-
-* Restore the npm packages:
-
-> yarn
+> dotnet fake run build.fsx -t "Fantomas-Git"
```

Comment:
This target also does the dotnet tool restore, so could remove that step.

Reply:
You can't run `dotnet fake` without first restoring the tools.
`Fantomas-Git` is also not all that one time, you would run it each time you want a newer commit of Fantomas.

Reply:
Ah yeah, derp, added that back in.